src/LibVLCSharp.Forms/Shared/PlaybackControls.xaml.cs

```diff
@@ -590,6 +588,21 @@ public bool IsAudioTracksSelectionButtonVisible
             set => SetValue(IsAudioTracksSelectionButtonVisibleProperty, value);
         }
 
+        /// <summary>
+        /// Identifies the <see cref="IsVideoTracksSelectionButtonVisible"/> dependency property.
+        /// </summary>
+        public static readonly BindableProperty IsVideoTracksSelectionButtonVisibleProperty = BindableProperty.Create(
```

Comment:
Is this code still relevant? Ditto for `IsAudioTracksSelectionButtonVisibleProperty` and `IsAudioTracksSelectionButtonVisible`

Reply:
Removed.